src/main/webapp/business.js

```diff
@@ -1,42 +1,70 @@
-// implementing map
+function getInfo() {
+  fetch('/business').then(response => response.json()).then((json) => {
+    var counter = 0;
+    for (var i = 1; i < json.length; i++) {
+      if (json[i][1] == 'in-person') {  // makes list of in-person businessess
+        var inPersonList = document.getElementById('inpersonbus');
```

Comment:
Maybe move this into a renderInPersonBLock or something like that.

Reply:
done!